server/data/courseClient.test.ts

```diff
@@ -67,6 +67,37 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
     })
   })
 
+  describe('createParticipation', () => {
+    const { prisonNumber } = person
+    const courseId = course1.id
+    const otherCourseName = 'Other course'
+
+    beforeEach(() => {
+      provider.addInteraction({
+        state: `A person exists with prison number ${prisonNumber}`,
+        uponReceiving: `A request to create a course participation for person "${prisonNumber}"`,
```

Comment:
I wonder if we need to have two tests here: one for passing a defined `courseId` and one for passing a defined `otherCourseName` (with the other `undefined` in each case)?